content/docs/instrumenting/clientlib_guidelines.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Client Library Guidelines
+sort_rank: 2
+---
+
+# Client Library Guidelines
+
+This document covers what functionality and API Prometheus client libraries should offer, with the aim of consistency across libraries, making the easy use cases easy and avoiding offering functionality that may lead users down the wrong path.
```

Comment:
We wrap lines at around 80 chars everywhere else in the docs, would be good to do the same here.